### PR TITLE
Add Documentation.md

### LINKED ISSUE #45: Documentation

### PR DESCRIPTION
**Pull request by bestape**

### CODE REVIEW

File: reviews/nft-trader/Documentation.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+## File Review
+### `Docs.docx`
+* Not .md file.
+* README.md is Git convention.
+* Did not open to review. 
+* The .docx format is unsafe, due to macros.
+* Could have created a VM in a GUI to review the .docx in a silo, but did not.
+* This review was done in a CLI ssh environment.
+* The .docx format is unusable in CLI ssh environments.
+
```
Comment (jlndnn) [inline reviews/nft-trader/Documentation.md:49]: Good points, I reviewed the docx as part of my PR so have a couple of points on it in addition
- Overall documentation is not comprehensive, main functions are briefly explained. I think this file is good for a some context and background on the project but I would not consider it a substitute for detailed technical documentation.
- There is a general lack of instructions, details of extra files needed for tests are vague. I think specific steps for running the code, running tests etc would have helped us

File: reviews/nft-trader/Documentation.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+---
+### `contracts/`
+#### `BatchSwap.sol`
+* Documentation could help with this, or function wrappers, but some of the for loops are functions without names. 
+* A comment with the same words as the name of the function it describes is unhelpful.
```
Comment (jlndnn) [inline reviews/nft-trader/Documentation.md:54]: I would agree with this - there are multiple examples of things like `struct swapStruct` commented with `// Swap Struct`. These could be removed. 

Overall the functions in this file are complex and could have benefitted from some inline explanations of the logical flow.

File: reviews/nft-trader/Documentation.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+* Uncertain about script flow, especially because there are multiple get functions.
+
+#### `TradeSquads.sol`
+* Trait array objects not given names.
+* Names like nRandRes are hard to understand.
+* For and while loops not given names.
+
```
Comment (jlndnn) [inline reviews/nft-trader/Documentation.md:61]: Notes on some of the other files
- BatchVault.sol : comments here are in Italian :)
- No comments on any of the below
  - AXBridge.sol
  - CKBridge.sol
  - ERC1155Five.sol
  - ERC1155Four.sol
  - ERC1155One.sol
  - ERC1155Six.sol
  - ERC1155Three.sol
  - ERC1155Two.sol

**Review by jlndnn — COMMENTED** [review]
Review looks great to me, I just had a couple of extra points on some files but all pretty minor

**Review by anthonygraignic — APPROVED** [review]

**Pull request merged**